it/java/gh/src/test/java/GHAPITest.java

```diff
@@ -0,0 +1,23 @@
+import apisdk.ApiClient;
+import com.microsoft.kiota.ApiException;
+import com.microsoft.kiota.authentication.AnonymousAuthenticationProvider;
+import com.microsoft.kiota.http.OkHttpRequestAdapter;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class BasicAPITest {
+
+    @Test
+    void basicTest() throws Exception {
+        var adapter = new OkHttpRequestAdapter(new AnonymousAuthenticationProvider());
+        adapter.setBaseUrl("http://127.0.0.1:1080");
+        var client = new ApiClient(adapter);
+
+        client.repos().byOwnerId("my-owner").byReposId("my-repo").get();
```

Comment:
@baywet here you have an Integration Test on the pattern we discovered in the GH API, this is not passing, but it shows the expected result (to me).

Reply:
I'm not sure why both are here? I'd expect only the first one?

Reply:
Because both of the endpoints should be easily and conveniently called, at least, this is the expected behavior to me.